### PR TITLE
Update CarrierLogo support Theming

### PR DESCRIPTION
**Pull request by williamkolmacka**
Added support for theming
Rewrited to styled components
Simplified css styling
Updated tests
Added flow strict type
Added optional type prop for carrier object
Added constants for carrier object type prop

### CODE REVIEW

File: src/CarrierLogo/__tests__/index.test.js [hunk 1]
```diff
@@ -2,7 +2,8 @@
 import * as React from "react";
 import { mount } from "enzyme";
```
Comment (wEprous) [inline src/CarrierLogo/__tests__/index.test.js]: Combination of `mount` and `ThemeProvider` is crazy. It generates a very large snapshot. I'll rewrite the test.

Comment (williamkolmacka) [reply src/CarrierLogo/__tests__/index.test.js]: okay, thanks

File: src/CarrierLogo/index.js [hunk 2]
```diff
@@ -1,121 +1,93 @@
...
 type Props = {
   size: "small" | "medium" | "large",
   carriers: Carrier[],
+  theme: typeof defaultTokens,
 };
 
-const CarrierLogo = (props: Props) => {
-  const { size, carriers } = props;
+const StyledCarrierLogo = styled.div`
+  border-radius: ${props => props.theme.borderRadiusNormal};
```
Comment (wEprous) [inline src/CarrierLogo/index.js]: Use object destructuring

File: src/CarrierLogo/index.js [hunk 3]
```diff
@@ -1,121 +1,93 @@
...
+`;
 
-  const sourceSize = carriers.length > 1 ? directorySizes.small : directorySizes[size];
-  const imageSize = carriers.length > 1 ? renderSizes.small : renderSizes[size];
-  const srcSetSize = carriers.length > 1 ? retinaSizes.small : retinaSizes[size];
+const CarrierLogo = (props: Props) => {
+  const { size, carriers, theme } = props;
+  const tokens = {
+    directorySizes: {
+      small: 16,
```
Comment (wEprous) [inline src/CarrierLogo/index.js]: Define constants for size prop

Comment (wEprous) [reply src/CarrierLogo/index.js]: Fixed.

File: src/CarrierLogo/index.js (hunk 2 above)
Comment (wEprous) [reply src/CarrierLogo/index.js]: Fixed.

File: src/CarrierLogo/CarrierLogo.js [hunk 4]
```diff
@@ -0,0 +1,88 @@
+// @flow
+import * as React from "react";
+import defaultTokens from "@kiwicom/orbit-design-tokens";
+import styled from "styled-components";
+
+import { SIZE_OPTIONS } from "./consts";
+import type { Props } from "./CarrierLogo";
+
+const baseURL = "//images.kiwi.com"; // token or const
```
Comment (tomashapl) [inline src/CarrierLogo/CarrierLogo.js]: Can you move this to `const.js` 🙏 ?

File: src/CarrierLogo/CarrierLogo.js.flow [hunk 5]
```diff
@@ -0,0 +1,14 @@
+// @flow
+import defaultTokens from "@kiwicom/orbit-design-tokens";
+
+import { SIZE_OPTIONS, CARRIER_TYPE_OPTIONS } from "./consts";
+
+type Carrier = { code: string, name: string, type?: $Values<typeof CARRIER_TYPE_OPTIONS> };
```
Comment (tomashapl) [inline src/CarrierLogo/CarrierLogo.js.flow]: $Values will not work, can you please specify it ? like "airline" | "bus" | "train"

File: src/CarrierLogo/CarrierLogo.js.flow [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+
+export type Props = {|
+  size?: $Values<typeof SIZE_OPTIONS>,
```
Comment (tomashapl) [inline src/CarrierLogo/CarrierLogo.js.flow]: Also here 🙏

**Pull request merged**